### DISCUSSION TITLE
杀毒软件查出源码有病毒

### DISCUSSION THREAD

**Discussion by YunzeWu** [Meloong-Git/PCL#3024] [讨论]
我下载源码时被查出有病毒。啊啊啊啊啊

**Comment by WForst-Breeze** [comment]
误报

**Comment by MoYuan-CN** [comment]
> 参考 #56 #666 和 PCL 内置帮助库：启动器 > 将 PCL 添加到 Windows 安全中心白名单

为 Microsoft Defender 和 360 等杀毒软件误报，请在 Windows 安全中心和杀毒软件中添加排除项或白名单，具体教程见 PCL 内置帮助库。

**Reply by MoYuan-CN** [reply]
欸 不对 源码？

**Reply by SqrtTan** [reply]
代码文件也会报病毒吗？

**Reply by SqrtTan** [reply]
反正我没听说过简单ascii文件报毒的

**Reply by allMagicNB** [reply]
> 反正我没听说过简单ascii文件报毒的

你确定你是说下载**源码**的时候报毒？

**Comment by allMagicNB** [comment]
因为用了VMProtect加壳防止反编译，系统检测到加壳，就觉得有嫌疑了
